demo-repository/exercises/demo/descr.html

```diff
@@ -3,7 +3,7 @@
 </p>
 
 <h2>The task</h2>
-
+<img src="/static/demo/files/OCaml.png">
```

Comment:
Adding some **alt**ernative text is generally a best practice:
```suggestion
<img src="/static/demo/files/OCaml.png" alt="OCaml">
```